etcdserver/cluster_util.go

```diff
@@ -51,14 +51,16 @@ func isMemberBootstrapped(cl *cluster, member string, tr *http.Transport) bool {
 // a response, or a Cluster cannot be successfully created from a received
 // response, an error is returned.
 func GetClusterFromRemotePeers(urls []string, tr *http.Transport) (*cluster, error) {
-	return getClusterFromRemotePeers(urls, true, tr)
+	return getClusterFromRemotePeers(urls, 0, true, tr)
```

Comment:
let set 5 seconds or higher here. it is much readable and understandable than 0.